pycbc/__init__.py

```diff
@@ -69,9 +69,9 @@ def sig_handler(signum, frame):
 try:
     # This is a crude check to make sure that the driver is installed
     try:
-        err = subprocess.call(["nvidia-smi"], stdout=open(os.devnull, 'wb'), stderr=open(os.devnull, 'wb'))
-        if err != 0:
-            raise ImportError("Cannot access 'nvidia-smi', driver may not be installed correctly")
+        loaded_modules = subprocess.check_output(["lsmod"], stderr=subprocess.STDOUT)
```

Comment:
Duncan, 

This has broken pycbc on python 2.6, so I've reverted this patch. Please modify the code to use a supported function such as check_call, or just call as before, and then resubmit.